cardano-node/src/Cardano/CLI/Tx.hs

```diff
@@ -34,7 +34,7 @@ import           Data.Text (Text)
 import qualified Data.Text as T
 import           Formatting ((%), sformat)
 
-import           Control.Tracer (traceWith, stdoutTracer)
+import           Control.Tracer (traceWith, nullTracer, stdoutTracer)
```

Comment:
do you still use 'stdoutTracer'?

Reply:
There're still some places in this module where `stdoutTracer` is used:

1. `issueGenesisUTxOExpenditure`
2. `issueUTxOExpenditure`
3. `nodeSubmitTx`